src/app/components/GalleryGrid/GalleryGrid.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import Image from "next/image";
+import { useLightbox } from "@/app/hooks/useLightbox";
+import { IconLoupe } from "@/app/icons/IconLoupe";
+import { galleryGridData } from "@/app/utils/galleryGridData";
```

Comment:
If galleryGridData is an array of images, it is worth ensuring that it is typed appropriately (e.g. as string[]). This can help identify type errors at the compilation stage, especially if you are using TypeScript.